pylhc/machine_settings_info.py

```diff
@@ -94,13 +98,18 @@ def _get_params() -> dict:
             help="List of knobnames. "
                  "If `None` (or omitted) no knobs will be extracted. "
                  "If it is just the string ``'all'``, "
-                 "all knobs will be extracted (can be slow)."
+                 "all knobs will be extracted (can be slow). "
                  "Use the string ``'default'`` the main knobs of interest."
```

Comment:
Feels like there's a missing word in here. Maybe "Use the string ``'default'`` for the main pre-defined knobs of interest."